rapids_build_backend/config.py

```diff
@@ -68,10 +68,14 @@ def __getattr__(self, name):
                         return str_val == "true"
                     return os.environ[env_var]
 
-                if config_name in self.config_settings:
+                try:
+                    value = self.config_settings[f"rapidsai.{config_name}"]
+                except KeyError:
+                    pass
+                else:
```

Comment:
IMO the if-else is preferable to a try-except here since in principle we have no idea how frequently users will be providing config settings to the wrapped backend. In general, try-excepts are relatively cheap in Python but they're still more expensive than an if-else if you expect a larger than miniscule exception rate. Aside from performance, which one you choose also conveys what your expectations are.

Reply:
Good point about how frequently the test fails. Updated.